pylint_plugins/api_models.py

```diff
@@ -53,11 +56,33 @@ def transform(cls):
             # Not a class we are interested in
             return
 
-        properties = schema.get('properties', {}).keys()
-
-        for property_name in properties:
+        properties = schema.get('properties', {})
+        for property_name, property_data in six.iteritems(properties):
             property_name = property_name.replace('-', '_')  # Note: We do the same in Python code
-            cls.locals[property_name] = [scoped_nodes.Class(property_name, None)]
+            property_type = property_data.get('type', None)
+
+            if isinstance(property_type, (list, tuple)):
+                # Hack for attributes with multiple types (e.g. string, null)
+                property_type = property_type[0]
+
+            if property_type == 'object':
```

Comment:
This is cool since pylint is now also aware of each API model attribute type.

It means we get improved static checking for API model members (e.g. if API model attribute type is defined as string and you want perform division on it, pylint will now throw since it knows the attribute type is string a not int / float).